source/components/triggering-changes-with-actions.md

```diff
@@ -0,0 +1,277 @@
+So far, you've learned how parent components can pass attributes in to a
+child component, and how that component can use those attributes from
+both JavaScript and its Handlebars template.
```

Comment:
I suggest dropping "Handlebars". Ember templates are simply Ember templates. They just happen to have Handlebars syntax.